### PR TITLE
fix(lint): Change endOfLine rules to better support linting on Windows

### PR DESCRIPTION
**Pull request by HowardBraham**
Before this PR, if you run eslint on Windows, you will get thousands of CRLF errors like this:
```
    1:52   error  Delete `␍`  prettier/prettier
    2:52   error  Delete `␍`  prettier/prettier
    3:74   error  Delete `␍`  prettier/prettier
```
You can fix them with `yarn lint:fix`, but they will probably come back again, depending on your git and text editor settings.

Meanwhile, git commits everything as LF, so it shouldn't affect Mac/Linux.

### CODE REVIEW

File: packages/base/src/environment.json [hunk 1]
```diff
@@ -209,6 +209,10 @@
       "name": "ClipboardEvent",
       "message": "This global is not available in the Node.js and browser environment."
     },
+    {
+      "name": "ClipboardItem",
```
Comment (mcmire) [inline packages/base/src/environment.json:213]: Were these changes also necessary to get things working on Windows or should these go in another PR?

**Review by mcmire — COMMENTED** [review]
I wonder why the default for `endOfLine` changed. It seems like something called "auto" should be the default. Oh well 🤷🏻‍♂️ 

Changes to the Prettier config look good, just had a question about the changes to `environment.json`.

File: packages/base/src/environment.json (hunk 1 above)
Comment (HowardBraham) [reply packages/base/src/environment.json:213]: Yeah good question.  That's just what happened when I ran `yarn generate`, but maybe it's not necessary?

Comment (mcmire) [reply packages/base/src/environment.json:213]: It's possible these files are out of date, but yeah, if the tests still pass here without these changes I'd say leave them out and we can address them in a new PR.

Comment (mcmire) [reply packages/base/src/environment.json]: Hmm, so when I run `yarn generate`, I don't get anything. Which Node version are you using?

Comment (HowardBraham) [reply packages/base/src/environment.json]: 16.20.0

Comment (mcmire) [reply packages/base/src/environment.json]: Ah - that might be the reason. The `.nvmrc` specifies v14, so that's the recommended development version of Node. Probably need to bump this, but in any case, I think that explains the difference in globals.

**Review by mcmire — APPROVED** [review]
Looks good!
